custom_model_runner/datarobot_drum/drum/runtime.py

```diff
@@ -48,12 +50,15 @@ def __exit__(self, exc_type, exc_value, exc_traceback):
         host = host_port_list[0]
         port = int(host_port_list[1]) if len(host_port_list) == 2 else None
 
-        run_error_server(host, port, exc_value)
+        with verbose_stdout(self.options.verbose):
+            run_error_server(host, port, exc_value)
 
         return False  # propagate exception further
 
 
 def run_error_server(host, port, exc_value):
+    logging.getLogger("werkzeug").setLevel(logging.ERROR)
```

Comment:
I'm not sure we should set it to error. If the user uses logger for their own logs, this'll hide them

Reply:
You are right.
For drum pipeline user can specify whatever log level is wanted.

So keep default INFO log level here won't harm I guess.